src/main/java/com/revature/model/Comment.java

```diff
@@ -30,116 +29,102 @@
  * 
  * @author Davin Merry
  */
+

```

Comment:
This blank line not needed between JavaDoc comments and thing being commented on.

Reply:
fixed